applications_tests/gls_sharp_navier_stokes_2d/flow_around_multiple_objects.output

```diff
@@ -0,0 +1,56 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       1024
+   Number of degrees of freedom: 3267
+   Volume of triangulation:      64
+Initial refinement around IB particles - Step : 1 of 2
+   Number of active cells:       1786
+   Number of degrees of freedom: 5688
+   Volume of triangulation:      64
+Initial refinement around IB particles - Step : 2 of 2
+   Number of active cells:       4456
+   Number of degrees of freedom: 14019
+   Volume of triangulation:      64
+
+*****************************************************************
+Steady iteration :        1/2
+*****************************************************************
++------------------------------------------+
+|  Force  summary particle 0               |
++------------------------------------------+
+particle_ID    T_z       f_x      f_y    
+          0 -0.295792 38.750700 3.913430 
++------------------------------------------+
+|  Force  summary particle 1               |
++------------------------------------------+
+particle_ID    T_z       f_x      f_y    
+          1 -0.322409 21.248258 8.361155 
++------------------------------------------+
+|  Force  summary particle 2               |
++------------------------------------------+
+particle_ID   T_z       f_x       f_y    
+          2 4.586123 39.240159 -8.988230 
```

Comment:
Why is the entire force history of the particles printed to the terminal at each iteration. This is far too verbose. It would be better to print a table with the force summary for all particles, and save to the files the history per particle. Like we do with the forces on the boundary conditions. Otherwise the terminal is flooded for no reason.

Reply:
It should be better now